SurfaceTopography/Uniform/ScalarParameters.py

```diff
@@ -52,30 +53,47 @@ def rms_height(topography, kind='Sq'):
         Root mean square height value.
     """
     n = np.prod(topography.nb_grid_pts)
-    # if topography.is_MPI:
     pnp = Reduction(topography._communicator)
     profile = topography.heights()
-    if kind == 'Sq':
-        return np.sqrt(
-            pnp.sum((profile - pnp.sum(profile) / n) ** 2) / n)
-    elif kind == 'Rq':
-        # Problem: when one of the processors holds the full data he isn't able
-        # to detect if any axis is MPI_Parallelized
-        # this problem is solved automatically if we do not support one axis
-        # to be zero
-        decomp_axis = [full != loc for full, loc in
-                       zip(np.array(topography.nb_grid_pts), profile.shape)]
-        temppnp = pnp if decomp_axis[0] else np
-        return np.sqrt(temppnp.sum(
-            (profile - temppnp.sum(profile, axis=0)
-             / topography.nb_grid_pts[0]) ** 2
-        ) / n)
-    else:
-        raise RuntimeError("Unknown rms height kind '{}'.".format(kind))
+
+    # Problem: when one of the processors holds the full data he isn't able
+    # to detect if any axis is MPI_Parallelized
+    # this problem is solved automatically if we do not support one axis
+    # to be zero
+    decomp_axis = [full != loc for full, loc in
```

Comment:
In #1   I say this is bogus. I suggest to remove mpi support for rms_height from profile. I think it is neither tested nor used.

Reply:
Okay, I've removed the parallelized code